veros/core/tke.py

```diff
@@ -13,8 +13,8 @@ def set_tke_diffusivities(state):
         tke_diff_out = set_tke_diffusivities_kernel(state)
         vs.update(tke_diff_out)
     else:
-        vs.kappaM = update(vs.kappaM, at[...], vs.kappaM_0)
-        vs.kappaH = update(vs.kappaH, at[...], vs.kappaH_0)
+        vs.kappaM = update(vs.kappaM, at[...], settings.kappaM_0)
+        vs.kappaH = update(vs.kappaH, at[...], settings.kappaH_0)
```

Comment:
This is the corresponding code path in PyOM.

```suggestion
        vs.kappaM = update(vs.kappaM, at[...], settings.kappaM_0)
        vs.kappaH = npx.where(vs.Nsqr[..., vs.tau] < 0.0, 1.0, settings.kappaH_0)
```

Reply:
In my opinion that should definitely be a toggle, but I really operate at very different scales than any of the models discussed here.

Reply:
Could you elaborate? We should definitely stay compliant to PyOM, but if you can make a case for setting kappaH unconditionally we could add a setting for this.